parcel_test.go

```diff
@@ -24,65 +27,148 @@ func getTestParcel() Parcel {
 		Client:    1000,
 		Status:    ParcelStatusRegistered,
 		Address:   "test",
-		CreatedAt: time.Now().UTC().Format(time.RFC3339),
+		Created_at: time.Now().UTC().Format(time.RFC3339),
 	}
 }
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+    if err != nil {
+        fmt.Println(err)
+        return
+    }
+    defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
-
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
 
+	var id int
+	id, err = store.Add(parcel)
+    require.NoError(t, err)
+	require.NotEmpty(t, id)
+
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
+	var returnParcel Parcel
+    returnParcel, err = store.Get(id)
+	require.NoError(t, err)
+	require.NotEmpty(t, returnParcel)
+    // Проверяем, что данные посылки совпадают с ожидаемыми
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+    assert.Equal(t, parcel.Client, returnParcel.Client, "Не совпадает")
+    assert.Equal(t, parcel.Status, returnParcel.Status, "Не совпадает")
+    assert.Equal(t, parcel.Address, returnParcel.Address, "Не совпадает")
+	pTime, _ := time.Parse(time.RFC3339, parcel.Created_at)
+	rTime, _ := time.Parse(time.RFC3339, returnParcel.Created_at)
+	assert.WithinDuration(t, pTime, rTime, time.Second, "Не совпадает")
+
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(id)
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
+	_, err = store.Get(id)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```